api/server.js

```diff
@@ -0,0 +1,45 @@
+// Imports
+const express = require("express");
+const helmet = require("helmet");
+const session = require("express-session");
+const SessionStore = require("connect-session-knex")(session);
+
+// Routes
+const userRoutes = require("../users/userRoutes.js");
+const registerRoutes = require("../users/registerRoutes.js");
+const loginRoutes = require("../users/loginRoutes.js");
+
+const server = express();
+const sessionConfig = {
+  name: "monkey",
+  secret: "super secret string",
+  resave: false,
+  saveUninitialized: false,
+  cookie: {
+    maxAge: 60 * 60 * 1000, // 1 hour. 60sec + 60 min + 1000 miliseconds
+    secure: false,
+    httpOnly: true
+  },
+  store: new SessionStore({
```

Comment:
good job on building out your server and configuring your session, calling in the middleware and setting up your routes.